server/common/utilities.js

```diff
@@ -6,4 +8,9 @@ const filter = f => object =>
     Object.entries(object).filter(([key, value]) => f(value, key, object))
   );
 
-module.exports = filter;
+const checkRole = (ids, userIdFromReq) => {
```

Comment:
I think also, that the first argument is a bit confusing, it should be named as `idsArray` to make this clear that we should pass array of ids here